specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -567,6 +860,98 @@
         }
       },
       "description": "The information of an AvailablePrivateEndpointType."
+    },
+    "PrivateDnsZoneGroup": {
+      "properties": {
+        "name": {
+          "readOnly": true,
```

Comment:
why is this needed?

Reply:
Not sure why, that would mean you couldnt write to it, which wouldn't allow puts. I can remove the readOnly tag, should I get rid of the one for type as well? will push once I have finished with other comments.